src/deployment/flavors.md

```diff
@@ -3,41 +3,158 @@ title: Creating flavors for Flutter
+at the end of each new configuration name. 
+
+5. Change the `development` Scheme to match the development 
+build configurations you’ve created to assign the correct build configurations 
+to the `development` scheme. 
+* In the Runner project, click Manage Schemes… a pop up window opens. 
+* Double click the development scheme. In the next step 
+(as shown in the screenshot), you’ll modify each scheme 
+to match its dev build configuration:
+
+![Step 5 Xcode image](/assets/images/docs/flavors/step5-ios-scheme.png){:width="100%"}
+
+## Using flavors in iOS
+
+Now that you’ve set up your development flavor, 
+you can add different product bundle identifiers per flavor. 
+A bundle identifier is a way to uniquely identify your application. 
+In this example, we want the Debug-development value to equal 
+com.flavor-test.dev. 
+
+1. Change the app bundle identifier to differentiate between schemes. 
+In **Product Bundle Identifier**, append `.dev` to each -development scheme value.            
+![Step 1 using flavors image.](/assets/images/docs/flavors/step1-using-flavors.png){:width="100%"}        
+2. In the Build Settings, set the Product Name value to match each flavor.        
+![Step 2 using flavors image.](/assets/images/docs/flavors/step2-using-flavors.png){:width="100%"}             
+3. Add the display name to info.plist.        
+Create a new key in info.plist called **Bundle Display Name** 
+with the value `$(PRODUCT_NAME)`. 
+![Step 2 using flavors image.](/assets/images/docs/flavors/step2-using-flavors.png){:width="100%"}    
+
+Now you have set up your development flavor by making a `development` scheme 
+in Xcode and setting the build configurations for that scheme. 
+
+For more information, skip to the [“Launching your app flavors”][] 
+section at the end of this document.
+
+## Using flavors in Android
+
+Setting up flavors in Android can be done in your projects 
+**build.gradle** file.
+
+1. In your IDE, inside your Flutter project, 
+navigate to **android** > **app** > **build.gradle**.   
+
+![Step 1 using flavors in Android image.](/assets/images/docs/flavors/android-step1.png){:width="100%"}    
+
+Add a flavors object with the specified environments along with values for 
+dimension, resValue, and applicationId. 
+The name of the application for each build is located in resValue 
+(as shown in the screenshot below from VSCode).
+## Setting up launch configurations
+
+If you are using VS Code as your IDE, the following steps 
+show you how to set up launch configurations. 
+Adding a **launch.json** file allows you to run the terminal command 
+`flutter run --flavor [environment name]`. 
+1. In the root directory of your project, add a folder called **.vscode**.    
+2. Inside the **.vscode** folder, create a file named **launch.json**.    
+3. In the **launch.json** file, add a configuration object for each flavor. 
+Each configuration has a **name**, **request**, **type**, **program**, 
+and **args** key.
```

Comment:
Question: Generally, only UNIX systems (like macOS and Linux) use dot-files. Is it .`vscode` on Windows, too? I suspect not. Make sure that these instructions work for Windows, too.
@jmagman

```suggestion
1. In the root directory of your project, add a folder called **.vscode**.    
2. Inside the **.vscode** folder, create a file named **launch.json**.    
3. In the **launch.json** file, add a configuration object for each flavor. 
    Each configuration has a **name**, **request**, **type**, **program**, 
    and **args** key.

You can now run the terminal command 
`flutter run --flavor [environment name]` or you can set up a run
configuration in your IDE. 
```

When you have a parenthetical sentence, you should put the parenthetical first, so they know they can stop reading if it doesn't interest or apply to them.